packages/cli/test/integration.test.ts

```diff
@@ -1366,15 +1367,12 @@ test('print the deploy help message', async t => {
   console.log(stdout);
   console.log(exitCode);
 
-  t.is(exitCode, 2);
-  t.true(stderr.includes(deployHelpMessage), `Received:\n${stderr}\n${stdout}`);
-  t.false(
-    stderr.includes('ExperimentalWarning'),
-    `Received:\n${stderr}\n${stdout}`
-  );
+  expect(exitCode).toBe(2);
+  expect(stderr.includes(deployHelpMessage)).toBe(true);
```

Comment:
```suggestion
  expect(stderr).toContain(deployHelpMessage)
```

Reply:
These were codemod changes I'll happily improve in a follow-up PR.